packages/next/server/dev/hot-reloader.ts

```diff
@@ -899,6 +899,10 @@ export default class HotReloader {
       )
     })
 
+    for (const key of Object.keys(entries)) {
```

Comment:
What is this intended to do?

Reply:
After stopping and starting the dev webpack instance we need to clear the previous entries or it causes the `on-demand-handler` to be out of sync with the active compilation. Can add a comment explaining this.